README.md

```diff
@@ -1,44 +1,88 @@
-Guides
-======
+# Guides
 
 [![Reviewed by Hound](https://img.shields.io/badge/Reviewed_by-Hound-8E64B0.svg)](https://houndci.com)
 
-Guides for getting things done, programming well, and programming in style.
-
-* [Best Practices](./best-practices)
-* [Code Review](./code-review)
-* [Tech Stack](./stack)
-* [How to](./how-to)
-* [Protocol](./protocol)
-  * [Communication](./protocol/communication)
-  * [Accessibility](./protocol/accessibility)
-  * [Git](./protocol/git)
-  * [iOS](./protocol/ios)
-  * [Open Source](./protocol/open-source)
-  * [Product Review](./protocol/product-review)
-  * [Rails](./protocol/rails)
-  * [React](./protocol/react)
-* [Security](./security)
-* [Style](./style)
-* [Web Performance](./web-performance)
-* [Working Together](./working-together)
-
-High level guidelines:
+Guides for working together, getting things done, programming well, and programming in style.
+
+### High level guidelines
 
 * Be consistent.
 * Don't rewrite existing code to follow this guide.
 * Don't violate a guideline without a good reason.
 * A reason is good when you can convince a teammate.
 
-A note on the language:
+### A note on the language
 
 * "Avoid" means don't do it unless you have good reason.
 * "Don't" means there's never a good reason.
 * "Prefer" indicates a better option and its alternative to watch out for.
 * "Use" is a positive instruction.
 
-Contributing
-------------
+## Guides by category
+
+[thoughtbot Tech Stack](/tech_stack.md)
```

Comment:
We'll need to add `-` to create bullets. The rich diff shows they're rendering as one long line of links.